cli/src/action/circuit/api.rs

```diff
@@ -275,6 +275,12 @@ impl fmt::Display for CircuitSlice {
             if let Some(public_key) = &member.public_key {
                 display_string += &format!("        Public Key: {}\n", public_key);
             }
+
+            display_string += &"        Endpoints:\n".to_string();
```

Comment:
This will result in lint when merged. Change this to 

```rust
display_string += "        Endpoints:\n";
```

Reply:
Updated